packages/ketcher-macromolecules/src/Editor.tsx

```diff
@@ -239,13 +242,32 @@ function Editor({ theme, togglerComponent }: EditorProps) {
                 sequenceNode.rnaBase.monomerItem,
               ];
 
+        const existingPreset = presets.find((preset) => {
+          const presetMonomers = [preset.sugar, preset.base, preset.phosphate];
+          return monomers.every((monomer, index) => {
+            return monomer?.props.Name === presetMonomers[index]?.props.Name;
+          });
+        });
+
+        let position: PresetPosition;
+        if (sequenceNode instanceof Nucleoside) {
+          position = 'chainEnd';
+        } else if (sequenceNode.sugar.attachmentPointsToBonds.R1 === null) {
+          position = 'chainStart';
+        } else {
+          position = 'chainMiddle';
```

Comment:
Please use constants or enum

Reply:
Changed to enum although type union with strings literals has all the benefits of enum as well – type checking, auto-complete and refactoring for all usages